### PR TITLE
Port some stuff from #2048 to snapshot

### PR DESCRIPTION
**Pull request by IotaBread**
This PR merges the things that aren't in the snapshot branch but are in #2048

### CODE REVIEW

File: mappings/net/minecraft/client/gui/screen/PresetsScreen.mapping [hunk 1]
```diff
@@ -1,4 +1,5 @@
 CLASS net/minecraft/class_430 net/minecraft/client/gui/screen/PresetsScreen
+	FIELD field_25043 RENAME_TO_LOGGER_OR_KICK_ME Lorg/apache/logging/log4j/Logger;
```
Comment (YanisBft) [inline mappings/net/minecraft/client/gui/screen/PresetsScreen.mapping]:
```suggestion
	FIELD field_25043 LOGGER Lorg/apache/logging/log4j/Logger;
```

File: mappings/net/minecraft/client/gui/screen/TitleScreen.mapping [hunk 2]
```diff
@@ -5,6 +5,7 @@ CLASS net/minecraft/class_442 net/minecraft/client/gui/screen/TitleScreen
 	FIELD field_17776 isMinceraft Z
 	FIELD field_18222 doBackgroundFade Z
 	FIELD field_19102 ACCESSIBILITY_ICON_TEXTURE Lnet/minecraft/class_2960;
+	FIELD field_23775 RENAME_TO_LOGGER Lorg/apache/logging/log4j/Logger;
```
Comment (YanisBft) [inline mappings/net/minecraft/client/gui/screen/TitleScreen.mapping]:
```suggestion
	FIELD field_23775 LOGGER Lorg/apache/logging/log4j/Logger;
```

File: mappings/net/minecraft/client/gui/screen/PresetsScreen.mapping (hunk 1 above)
Comment (IotaBread) [reply mappings/net/minecraft/client/gui/screen/PresetsScreen.mapping]: I did it again ~~please kick me~~

**Comment by IotaBread** [comment]
2 commits because, for some reason, one of the suggested changes wasn't added to the batch

**Review by YanisBft — APPROVED** [review]

**Review by liach — APPROVED** [review]

**Comment by YanisBft** [comment]
Marking as ready to merge as the same mappings have been approved in the other PR

**Pull request merged**